packages/ui/src/components/Visualization/Custom/hooks/delete-hotkey.hook.tsx

```diff
@@ -0,0 +1,30 @@
+import { useCallback, useEffect } from 'react';
+import hotkeys from 'hotkeys-js';
+import { useDeleteStep } from './delete-step.hook';
+import { IVisualizationNode } from '../../../../models';
+
+export default function useDeleteHotkey(selectedVizNode: IVisualizationNode | undefined, clearSelected: () => void) {
+  const { onDeleteStep } = useDeleteStep(selectedVizNode);
+  const handleKeyDown = useCallback(() => {
```

Comment:
Unfortunately, I don't think this will be enough, as the caller will always generate a new function. I think it would be better to use `useCallback` outside and leave this one as it comes.